README-MULTISITE.md

```diff
@@ -1,153 +1,511 @@
 
-**Note:** `rgw_zonemaster` should have the value of `false` and `rgw_zonesecondary` should be `true`
+```
+rgw_zonemaster: false
+rgw_zonesecondary: true
+rgw_zonegroupmaster: true
+rgw_multisite_proto: "http"
+rgw_realm: 'usa'
+rgw_zonegroup: 'alaska'
+rgw_zone: 'juneau'
+radosgw_address: "{{ _radosgw_address }}"
+radosgw_frontend_port: 8080
+rgw_zone_user: edward.lewis
+rgw_zone_user_display_name: "Edward Lewis"
+system_access_key: yu17wkvAx3B8Wyn08XoF
+system_secret_key: 5YZfaSUPqxSNIkZQQA3lBZ495hnIV6k2HAz710BY
+rgw_pull_proto: http
+rgw_pull_port: 8080
+rgw_pullhost: cluster0-rgw-001-hostname
+```
+
+6. Run the ceph-ansible playbook on your secondary cluster.
+
+## Conclusion
+
+There will be 2 realms in this configuration, `france` and `usa`, with RGWs and RGW zones in both clusters. 
+
+# Deployment Scenario #4: Multiple Realms over Multiple Ceph Clusters with Multiple Instances
 
-**Note:** The endpoint made from `rgw_pull_proto` + `rgw_pull_host` + `rgw_pull_port` for each realm should be resolvable by the Primary Ceph clusters mons and rgws
+More than 1 RGW can be running on a single host. To configure multisite for a host with more than one rgw instance running on the host, `rgw_instances` and `radosgw_num_instances` must be configured.
 
-**Note:** `rgw_zone_user`, `system_access_key`, and `system_secret_key` should match what you used in the Primary Cluster
+Each item in `rgw_instances` (declared in a host_vars file) represents an RGW on that host. In each item is the multisite configuration for that RGW. 
 
-6. **(Optional)** Edit the rgws.yml in group_vars for rgw related pool creation
+Here is an example:
 
 ```
-rgw_create_pools:
-  "{{ rgw_zone }}.rgw.buckets.data":
-    pg_num: 64
-    size: ""
-    type: ec
-    ec_profile: myecprofile
-    ec_k: 5
-    ec_m: 3
-  "{{ rgw_zone }}.rgw.buckets.index":
-    pg_num: 8
-    size: ""
-    type: replicated
+rgw_instances:
+  - instance_name: 'rgw0'
+    rgw_realm: 'usa'
+    rgw_zonegroup: 'alaska'
+    rgw_zone: 'juneau'
+    radosgw_address: "{{ _radosgw_address }}"
+    radosgw_frontend_port: 8080
+    rgw_zone_user: edward.lewis
+    rgw_zone_user_display_name: "Edward Lewis"
+    system_access_key: yu17wkvAx3B8Wyn08XoF
+    system_secret_key: 5YZfaSUPqxSNIkZQQA3lBZ495hnIV6k2HAz710BY
 ```
-**Note:** The pg_num values should match the values for the rgw pools created on the primary cluster. Mismatching pg_num values on different sites can result in very poor performance.
 
-**Note:** An online pgcalc tool (ex: https://ceph.io/pgcalc) should be used to determine the optimal sizes for the rgw.buckets.data, rgw.buckets.index pools as well as any other pools declared in this dictionary.
+Here is an example of a host_vars for a host (ex: rgw-000 in the examples) containing 2 rgw_instances:
 
-7. Run the ceph-ansible playbook on your 2nd cluster
+```
+rgw_zonemaster: true
+rgw_zonesecondary: false
+rgw_zonegroupmaster: true
+rgw_multisite_proto: "http"
+radosgw_num_instances: "2"
```

Comment:
you don't need to specify `radosgw_num_instances` when providing the rgw_instances list manually.

We set this in the CI configuration only for test purpose.